launch_xml/launch_xml/entity.py

```diff
@@ -36,7 +36,7 @@ def __init__(
         *,
         parent: 'Entity' = None
     ) -> Text:
-        """Construnctor."""
+        """Constructor."""
```

Comment:
It says this needs to be "in an imperative mood"

Maybe "Construct the Entity"